packages/react-environment/src/types/chain-config.interface.ts

```diff
@@ -11,5 +11,5 @@ export interface ChainConfig {
   blockExplorerStub?: string;
   logo: React.ComponentType;
   nativeCurrencyId: WebbCurrencyId;
-  currencies: Array<{ currencyId: WebbCurrencyId; enabled: boolean; address: string }>;
```

Comment:
I like cleaning this up but to be just `WebbCurrenchID` however, it seems the enabled key could be useful to disable the asset for some reason, WTDYT?